systemverilog-plugin/UhdmAst.cc

```diff
@@ -1350,14 +1349,15 @@ void UhdmAst::move_type_to_new_typedef(AST::AstNode *current_node, AST::AstNode
             delete type_node;
         } else {
             type_node->str = "$enum" + std::to_string(shared.next_enum_id());
-            for (auto *enum_item : type_node->children) {
-                enum_item->attributes["\\enum_base_type"] = AST::AstNode::mkconst_str(type_node->str);
-            }
             auto wire_node = new AST::AstNode(AST::AST_WIRE);
             wire_node->is_reg = true;
             wire_node->attributes["\\enum_type"] = AST::AstNode::mkconst_str(type_node->str);
             if (!type_node->children.empty() && type_node->children[0]->children.size() > 1) {
                 wire_node->children.push_back(type_node->children[0]->children[1]->clone());
+            } else {
+                // Add default range
+                auto range = make_range(31, 0);
+                wire_node->children.push_back(range);
```

Comment:
Small nit, but I think it looks better.
```suggestion
                wire_node->children.push_back(make_range(31, 0));
```

Reply:
Changed.